<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="3" hostname="DESKTOP-BNERN3A" failures="2" tests="3" name="dynamic test execution" time="14.034" errors="0" timestamp="2024-06-02T23:16:33 AEST">
  <testcase classname="appiummavenproject.TestCases" name="fourthtest" time="4.515">
    <failure type="java.lang.NullPointerException" message="Cannot invoke &amp;quot;com.aventstack.extentreports.ExtentTest.addScreenCaptureFromBase64String(String)&amp;quot; because &amp;quot;this.etest&amp;quot; is null">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "com.aventstack.extentreports.ExtentTest.addScreenCaptureFromBase64String(String)" because "this.etest" is null
at appiummavenproject.TestCases.fourthtest(TestCases.java:113)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at appiummavenproject.Triggeringprogram.run(Triggeringprogram.java:118)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 38 stack frames]]>
    </failure>
  </testcase> <!-- fourthtest -->
  <testcase classname="appiummavenproject.TestCases" name="fifthtest" time="3.934">
    <failure type="java.lang.NullPointerException" message="Cannot invoke &amp;quot;com.aventstack.extentreports.ExtentTest.addScreenCaptureFromBase64String(String)&amp;quot; because &amp;quot;this.etest&amp;quot; is null">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "com.aventstack.extentreports.ExtentTest.addScreenCaptureFromBase64String(String)" because "this.etest" is null
at appiummavenproject.TestCases.fifthtest(TestCases.java:130)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at appiummavenproject.Triggeringprogram.run(Triggeringprogram.java:118)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 38 stack frames]]>
    </failure>
  </testcase> <!-- fifthtest -->
  <testcase classname="appiummavenproject.TestCases" name="sixthtest" time="5.565"/>
  <testcase classname="appiummavenproject.TestCases" name="secondtest" time="0.0">
    <ignored/>
  </testcase> <!-- secondtest -->
  <testcase classname="appiummavenproject.TestCases" name="thirdtest" time="0.0">
    <ignored/>
  </testcase> <!-- thirdtest -->
  <testcase classname="appiummavenproject.TestCases" name="firsttest" time="0.0">
    <ignored/>
  </testcase> <!-- firsttest -->
</testsuite> <!-- dynamic test execution -->
